### ISSUE TITLE
Disable individual items in a MultiSelect

### ISSUE THREAD

**Issue by jamtrott** [equinor/design-system#1430]
We are developing a tool, where we use a MultiSelect component to allow users to select one or more 'items' to be plotted in a figure. For various reasons, some of the items that may be selected in the MultiSelect do not have any data available for plotting. Thus, if the user selects such an item, nothing is plotted.

Now, we could just remove these "useless" items from the list that we pass to the MultiSelect component. However, for our use case, it is important for the user to be made aware that these items do exist, but that their data is missing. Therefore, we would instead like to keep the items in the dropdown list shown by the MultiSelect component, but make it so that they are not selectable. Perhaps these items could be shown as being "disabled" in some way (e.g., greyed out text).

To summarise, I would like to disable the selection of individual items in a MultiSelect component.

**Comment by mimarz** [comment]
Thanks for reporting this feature request! 

We'll have a look at it on todays planning meeting 📝

**Comment by mimarz** [comment]
The best way to introduce a feature like this is to introduce a data-model for Single/Multi-Select. Preceding issue #1416
